packages/components/src/translations/de.ts

```diff
@@ -37,7 +37,12 @@ const translation: Translation = {
   selectAColorFromTheScreen: 'Farbe vom Bildschirm auswählen',
   showPassword: 'Passwort anzeigen',
   slideNum: slide => `Folie ${slide}`,
-  toggleColorFormat: 'Farbformat umschalten'
+  toggleColorFormat: 'Farbformat umschalten',
+  badge_danger: 'Gefahr',
+badge_neutral: 'Neutral',
+badge_primary: 'Primär',
+badge_success: 'Erfolg',
+badge_warning: 'Warnung',
```

Comment:
Translation keys shouldn't be component specific, as long as there translated term doesn't contain the component (e. g. `Close menu` could lead to the key `closeMenu` or similar).

Furthermore the texts should inform the user approriately, "Primär" doesn't give any information to screenreaders. Therefore some mapping has to be done on component side.

Here's my proposal, feel free to merge into your branch: https://github.com/synergy-design-system/synergy-design-system/pull/394

See especially https://github.com/synergy-design-system/synergy-design-system/pull/394/files#diff-0899acde5808332ea69599190c82ad6208b1704765348446d27c885f5f33ae01R52 and https://github.com/synergy-design-system/synergy-design-system/pull/394/files#diff-6e75b3b7229331821ff3bc44f3240f7385fad9d7d38c30f7094a92e1f8168890R42